data/RWTH-NFDI-Jupyter-Service.yaml

```diff
@@ -0,0 +1,42 @@
+title: RWTH Aachen University
+provider: HPC.ITC.RWTH
+service_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/
+support: servicedesk@itc.rwth-aachen.de
+health_api_url: https://jupyterhub.hpc.itc.rwth-aachen.de:9651/hub/api/health
+documentation_url: https://help.itc.rwth-aachen.de/service/rhr4fjjutttf/article/689934fec5a34c909c54606f6bc2e827/
+target_group_open_for: Researchers, RWTH Students/Employees/Guests, Juelich Members, Users sponsored by other Users, Users upon project application 
+login_process: Login with RWTH HPC Account through most NHR or NRW VPNs
+features:
+  version: JupyterHub 3.0+, JupyterLab 3.4.7+
+  kernels: ["Python 3.9", "Octave", "R", "C++ 11/14/17", "Matlab 2022b", "Matlab 2020b",  "Python: Keras", "Python: Tensorflow",  "Python: Tensorboard", "Python: Gurobi", "Python: Pytorch" ]
+  extensions: ["ipytree",
+        "jupyterlab-drawio",
+        "jupyterlab-plotly",
+        "jupyterlab_iframe",
+        "nbdime-jupyterlab",
+        "jupyter-matplotlib",
+        "jupyterlab_pygments",
+        "jupyterlab/git",
+       " jupyterlab/latex",
+        "jupyter-widgets/jupyterlab-manager"]
+  proxy_apps: ["Matlab 2022b", "Matlab 2020b"]
+  install: false
+  shared_folder: true
+  persistent_storage: true
+  misc: ["Available core-hours vary based on user affiliation or project.", "Runs on HPC Hardware", "Users installs planned, by request only at the moment."]
+resources:
+  default_server_user: 1
+  max_server_user: 5
+  # cpu values refer to vCPUs and are guarantees in case of contention, limits are 8x higher
+  default_cpu: 1
+  max_cpu: 48
+  default_cpu_time: 4h
+  max_cpu_time: 172 h
+  default_memory: 4 GB
+  max_memory: 185 GB
+  default_gpu: 0
+  max_gpu: 2
+  default_disk: 150 GB /home + 250 GB /work  
+  max_disk: 1 TB /hpcwork  
+  default_persistent_disk: 150 GB /home + 250 GB /work   
+  max_persistent_disk: 1 TB /hpcwork 
```

Comment:
This would be an invalid format for range queries... Is it important for a user to understand (at this point) the difference between the various paths (/home, /work, /hpcwork) ? If not, just sum up the numbers and skip the paths.

Reply:
Ok.